src/modules/tasks/task-list/tasks.html

```diff
@@ -78,7 +78,7 @@ <h1 class="bold-text">Tasks</h1>
                     <i ng-class="{ 'pficon pficon-error-circle-o': (task.status==='failed' ), 'pficon pficon-ok': (task.status==='finished' ), 'pficon pficon-warning-triangle-o': (task.status==='warning' ), 'fa fa-spinner': (task.status==='processing' || task.status==='new' )} "></i>
                 </div>
                 <div class="ft-column ft-main list-view-pf-additional-info-item list-view-pf-additional-info-item-stacked task-flow ">
-                    <a href="" class="bold-text " ng-click="taskCntrl.goToTaskDetail(task.job_id) " ng-show="task.flow ">{{task.flow}}</a>
+                    <b>{{task.flow}}</b>
```

Comment:
@a2batic You need to add condition for ng-show="task.flow"

Reply:
@gnehapk, will add this